### PR TITLE
[SUPPLY] Blueshield Crates

### PR DESCRIPTION
**Pull request by KillianKirilenko**
Adds a pair of locked crates with replacement Blueshield gear to cargo, under the Misc section. One contains some job-specific clothing, the other contains replacement weapon kits. Both require `access_heads` to unlock.

Allows for missing gear to be replaced or weapons to be changed without admin intervention, in the event of items being lost, destroyed, or taken off-shift.

### CODE REVIEW

**Comment by TheFurryFeline** [comment]
Do both listings from _vr and _yw versions still show on the list? Unless I'm remembering wrong, the overrides were being finicky last I tried to do something like that and not everything was showing.

**Comment by TheFurryFeline** [comment]
Never mind, tested myself and seems they're all listed onto the console under miscellaneous.Nice.

**Comment by KillianKirilenko** [comment]
Yeah, I put them under Misc together since they're not Security gear and there didn't seem to be any other category that really fit both of them at once.

I test all of my PRs locally before I make them, even if it's simple stuff like this.

**Comment by TheFurryFeline** [comment]
:+1:

File: code/datums/supplypacks/misc_yw.dm [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	contains = list(
+			/obj/item/clothing/head/beret/blueshield = 1,
+			/obj/item/clothing/under/yw/blueshield = 1,
+			/obj/item/clothing/under/yw/blueshield2 = 1,
+			/obj/item/clothing/suit/armor/yw/blueshield = 1,
+			/obj/item/clothing/suit/armor/yw/blueshieldcoat = 1,
+			/obj/item/clothing/accessory/poncho/roles/cloak/blueshield = 1,
+			/obj/item/clothing/accessory/holster/leg = 1
+			)
+	cost = 50
```
Comment (izac112) [inline code/datums/supplypacks/misc_yw.dm]:
```suggestion
	cost = 80
```
Up to 80 to discourage this from being bought unless absolutely needed.

File: code/datums/supplypacks/misc_yw.dm [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	containername = "Blueshield equipment"
+	access = access_heads
+	
+/datum/supply_pack/misc/blueshieldweapons
+	name = "Blueshield Weapon Kits"
+	contains = list(
+			/obj/item/gunbox/blueshield = 1,
+			/obj/item/gunbox/blueshield/secondary = 1
+			)
+	cost = 50
```
Comment (izac112) [inline code/datums/supplypacks/misc_yw.dm]:
```suggestion
	cost = 100
```
Up to 100 this **should** not be needed 99 times out of 100 and if it is the blueshield probably should be demoted or at least reprimanded, plus it discourages anyone else from ordering blueshield gear they should not have their hands on. 

Plus the blueshield gun boxes have access to some powerful stuff and a gun you cant get your hand on any other way.

Comment (SplinterGP) [reply code/datums/supplypacks/misc_yw.dm]: Being 50 sounds very abusable since the blueshield guns are fairly unique and strong

Comment (KillianKirilenko) [reply code/datums/supplypacks/misc_yw.dm]: Cost increase is definitely reasonable, but I'd argue for different reasons than just the weapons themselves being unique/strong; the cases allow you to pick from a wider range of weapons and they come with a decent amount of free ammo off the bat. Committing now.

**Comment by RadiantFlash7** [comment]
I like the idea of the blueshield being able to replace gear, especially if there was blueshields on the shift prior. My one concern would be non-blueshields ordering the weapons crate for the special weapons- BUT that can be allievated with an SOP change as well.

**Comment by KillianKirilenko** [comment]
I think an easy additional way to cut down on that would also be to add a new `access_blueshield` level. Currently, `access_heads` includes the RD & CMO (who *really* shouldn't have either of the weapon cases) as well as IAAs (who **definitely** shouldn't!). I'm not particularly comfortable with that idea, but it was the best-fit and I didn't want to add a new access level straight up.

**Comment by izac112** [comment]
https://www.yawn.ocry.com/Standard_Operating_Procedure#Blueshield_Equpiment

**Comment by izac112** [comment]
We probably should give blueshield their own access in the future yes.

**Pull request merged**